tests/integration/interactive_add_binding_test.go

```diff
@@ -224,12 +224,12 @@ var _ = Describe("odo add binding interactive command tests", func() {
 			var nsWithNoService string
 
 			BeforeEach(func() {
+				nsWithNoService = commonVar.CliRunner.CreateAndSetRandNamespaceProject()
```

Comment:
Why this new variable? The operators we're using will be installed in all the namespaces.

Reply:
This was added as part of #5936 (to specifically test service binding from a namespace with no services). From the diff, I can see that @anandrkskd has just moved the line to be called earlier, but I am not sure why this line was moved..